### PR TITLE
Add mysql pvc example

### PR DESCRIPTION
**Pull request by alexander-demicev**
This example is failing on backup creation, I don't know if it's my fault or ark.

### CODE REVIEW

File: roles/pvc/mysql_pvc/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  retries: 20
+  delay: 30
+  ignore_errors: yes
+  vars:
+    pod: '{{ lookup("k8s",
+      kind="Pod",
+      namespace="mysql-persistent",
+      label_selector="deployment=mysql-1"
+    )}}'
+
```
Comment (dymurray) [inline roles/pvc/mysql_pvc/tasks/main.yml:108]: Can we add a check to verify the persistent volume was restored?

File: roles/pvc/mysql_pvc/files/mysql-persistent-template.yml [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: mysql-persistent
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  annotations:
+    template.openshift.io/expose-database_name: "{.data['database-name']}"
```
Comment (jwmatthews) [inline roles/pvc/mysql_pvc/files/mysql-persistent-template.yml]: What do the template.openshift.io annotations provide in this example?

I had assumed they'd be used for 'OpenShift Templates' but not seeing an OpenShift Template being used here.

Comment (alexander-demicev) [reply roles/pvc/mysql_pvc/files/mysql-persistent-template.yml]: You are right, I'll remove it.

File: roles/pvc/mysql_pvc/tasks/main.yml (hunk 1 above)
Comment (alexander-demicev) [reply roles/pvc/mysql_pvc/tasks/main.yml:108]: Yes, I'll add it there. Do you think we would also need to check all other resources in this example?

Comment (dymurray) [reply roles/pvc/mysql_pvc/tasks/main.yml:108]: At the least I think we need the PV for this case specifically. As long as other test cases cover the other resources I'm looking for each example to cover some type of corner case the others dont. Since services/routes are used frequently they don't need to be checked in every test case, preferably we are testing the things that make each test unique.

**Comment by alexander-demicev** [comment]
@dymurray @jwmatthews PR is ready for another review :)

File: README.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 Ansible automation to load test-data of various sample applications to aid OCP Migration development efforts.
 
 # Usage
-For backups/restores ark is required, it can be install using playbooks from this repo ```https://github.com/fusor/ocp-velero-ansible```
+Ark is required for backups/restores, it can be installed using playbooks from this repo ```https://github.com/fusor/ocp-velero-ansible```
```
Comment (jwmatthews) [inline README.md]: I'd suggest to use the term 'Velero' on anything new

Comment (alexander-demicev) [reply README.md]: Sure, the new name just doesn't stick :)

**Review by jwmatthews — APPROVED** [review]
